<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Order</class>
 <widget class="QWidget" name="Order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>345</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
border-radius: 8px;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>345</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border-radius: 8px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Total_L_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>315</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 106, 110);
font: 600 14pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: left;
</string>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Orderid_PB">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>42</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: left;</string>
    </property>
    <property name="text">
     <string>#order id</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	background: white;
	color: rgb(81, 81, 81);
	font: 600 9pt &quot;Segoe UI&quot;;
border: none;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Qty_L">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: white;
	color: rgb(81, 81, 81);
	font: 600 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Qty</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QScrollArea {
    background-color: transparent;
    border: none;
    padding: 0px;
}

QScrollArea &gt; QWidget &gt; QWidget {
    background-color: transparent;
    border: none;
}

QScrollArea &gt; QWidget &gt; QScrollBar:vertical {
    background: rgb(223, 229, 229); 
    width: 4px;
    padding-bottom: 5px;
    padding-top: 5px;
}

QScrollArea &gt; QWidget &gt; QScrollBar::handle:vertical {
    background:  rgb(255, 137, 64);
    min-height: 3px;
    border-radius: 2px;
}

QScrollArea &gt; QWidget &gt; QScrollBar::add-line:vertical {
    background: rgb(223, 229, 229);
}

QScrollArea &gt; QWidget &gt; QScrollBar::sub-line:vertical {
    background: rgb(223, 229, 229);
}

QScrollArea &gt; QWidget &gt; QScrollBar::add-page:vertical, QScrollArea &gt; QWidget &gt; QScrollBar::sub-page:vertical {
    background: rgb(223, 229, 229);
}

QScrollArea &gt; QWidget &gt; QScrollBar::horizontal {
    background: rgb(223, 229, 229); 
    height: 0px;
}

QMenu {
    border: 1px solid white;
    border-radius: 5px;
    background-color: rgb(6, 106, 110);
    padding: 7px;
    color: white;
}

QMenu::item {
    padding: 4px;
}

QMenu::item:hover {
    border: none;
    outline: none;
}

QMenu::item:selected {
    background-color: rgb(255, 137, 64);
    border-radius: 5px;
    color: rgb(31, 48, 58);
}

QMenu::separator {
    height: 1px;
    background-color: white;
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>309</width>
          <height>179</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Orderstatus_L">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>42</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: right;</string>
    </property>
    <property name="text">
     <string>order stat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>315</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 14pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: left;</string>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="Price_L">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: white;
	color: rgb(81, 81, 81);
	font: 600 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>290</width>
      <height>1</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
    height: 0.5px;
    border: none;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Order_L">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>40</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 9pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: left;
background: white;</string>
    </property>
    <property name="text">
     <string>Order: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Customerinfo_PB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>11</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 11pt &quot;Segoe UI&quot;;
border: none;
padding-bottom: 3px;
text-align: left;</string>
    </property>
    <property name="text">
     <string>Name - LastName</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
	background: white;
	color: rgb(81, 81, 81);
	font: 600 9pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="displayFormat">
     <string>ddd, yyyy MMMM dd</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Table_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(6, 106, 110);
border: none;
border-radius: 12px;
font: 700 15pt &quot;Segoe UI&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>A4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="Paystatus_CB">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>11</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background: rgb(255, 204, 153);
	color: rgb(31, 48, 58);
	border: none;
	border-radius: 6px;
	font: 600 10pt &quot;Segoe UI&quot;;
	padding-right: 6px;
}

QCheckBox::indicator {
	background: transparent;
	border: none;
   width: 16px;
   height: 16px;
}</string>
    </property>
    <property name="text">
     <string>Pending</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Items_L">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: white;
	color: rgb(81, 81, 81);
	font: 600 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>308</y>
      <width>290</width>
      <height>1</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>179</red>
          <green>179</green>
          <blue>179</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
    height: 0.5px;
    border: none;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <zorder>Total_L_2</zorder>
   <zorder>Orderid_PB</zorder>
   <zorder>timeEdit</zorder>
   <zorder>Qty_L</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>Orderstatus_L</zorder>
   <zorder>Total_L</zorder>
   <zorder>Price_L</zorder>
   <zorder>line</zorder>
   <zorder>Order_L</zorder>
   <zorder>Customerinfo_PB</zorder>
   <zorder>dateEdit</zorder>
   <zorder>Paystatus_CB</zorder>
   <zorder>Items_L</zorder>
   <zorder>line_2</zorder>
   <zorder>Table_L</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
